bot/resources/tags/generators.md

```diff
@@ -1,5 +1,10 @@
 **Python Generators**
 
+A generator is a function that returns a sequence of values,
+producing them on demand, i.e. without storing them in a list.
+
+---
```

Comment:
Discord doesn't render these either but it still kind of works as a separator I guess.